### PR TITLE
[IMP] accounting/l10n_co: additional context

### PR DESCRIPTION
**Pull request by samueljlieber**
This PR improves the content of the Colombian localization documentation by adding additional context to most sections per https://github.com/odoo/documentation/pull/4425.

This PR can be FWP with small edits needed for 15.0 and 16.0.

### CODE REVIEW

**Review by dade-odoo — CHANGES_REQUESTED** [review]
Hey @samueljlieber - your updates look good. I made a few comments/edits to the doc in general. Feel free to reach out to discuss any of them if you want :)

**Comment by samueljlieber** [comment]
Hi @dade-odoo thank you for your review and suggestions for improvement! I implemented them in https://github.com/odoo/documentation/commit/48ba2f857651f326d5efb04a80a84483c6623c83.

**Comment by dade-odoo** [comment]
Hey @samueljlieber - I'll go ahead and approve this and pass it on to @odoo/be-doc-review, but I am still seeing a typo on line 89 - "modules **are** installed" or "module's installed" seem more appropriate to me :) I'll let you handle it

**Review by dade-odoo — APPROVED** [review]

**Comment by samueljlieber** [comment]
Hi @dade-odoo you are right, thank you for catching that! I just pushed a commit changing line 89 to "modules are installed" in https://github.com/odoo/documentation/commit/b433243cf1d467bf40046d6f8ac94e54c1e34bbd

**Comment by samueljlieber** [comment]
Hi @odoo/be-doc-review this PR is ready for your review 🙂

File: content/applications/finance/fiscal_localizations/colombia.rst [hunk 1]
```diff
@@ -36,74 +37,78 @@ localization:
...
      - Default :ref:`fiscal localization package <fiscal_localizations/packages>`. This module adds
        the base accounting features for the Colombian localization: chart of accounts, taxes,
-       withholdings, identification document type.
+       withholdings, and identification document type.
    * - :guilabel:`Colombian - Accounting Reports`
      - `l10n_co_reports`
      - Includes accounting reports for sending certifications to suppliers for withholdings applied.
    * - :guilabel:`Electronic invoicing for Colombia with Carvajal`
      - `l10n_co_edi`
      - This module includes the features that are required for the integration with Carvajal and
```
Comment (larm-odoo) [inline content/applications/finance/fiscal_localizations/colombia.rst:46]:
```suggestion
     - This module includes the features required for integration with Carvajal, and
```

File: content/applications/finance/fiscal_localizations/colombia.rst [hunk 2]
```diff
@@ -36,74 +37,78 @@ localization:
...
-       generates the electronic invoices and support document related to the vendor bills based on
+       generates the electronic invoices and support document related to the vendor bills, based on
```
Comment (larm-odoo) [inline content/applications/finance/fiscal_localizations/colombia.rst:47]:
```suggestion
       generates the electronic invoices and support documents related to the vendor bills, based on
```

File: content/applications/finance/fiscal_localizations/colombia.rst [hunk 3]
```diff
@@ -36,74 +37,78 @@ localization:
...
        |DIAN| regulations.
    * - :guilabel:`Colombian - Point of Sale`
      - `l10n_co_pos`
-     - Includes Point of Sale Receipt for Colombian Localization.
+     - Includes Point of Sale receipt for Colombian localization.
```
Comment (larm-odoo) [inline content/applications/finance/fiscal_localizations/colombia.rst:51]:
```suggestion
     - Includes Point of Sale receipts for Colombian localization.
```

Comment (larm-odoo) [reply content/applications/finance/fiscal_localizations/colombia.rst:51]: "receipts" sounds better- UNLESS there is only a single receipt produced (but I doubt it).

File: content/applications/finance/fiscal_localizations/colombia.rst [hunk 4]
```diff
@@ -36,74 +37,78 @@ localization:
...
 ---------------------
 
-To configure your company information, go to the :menuselection:`Contacts` app and search for your
-company. Alternatively, activate :ref:`developer mode <developer-mode>` and navigate to
+To configure your company information, go to the :menuselection:`Contacts` app, and search for your
+company.
+
+Alternatively, activate :ref:`developer mode <developer-mode>` and navigate to
 :menuselection:`General Setting --> Company --> Update Info --> Contact`. Then, edit the contact
 form to configure the following information:
```
Comment (larm-odoo) [inline content/applications/finance/fiscal_localizations/colombia.rst:65]:
```suggestion
form and configure the following information:
```

File: content/applications/finance/fiscal_localizations/colombia.rst [hunk 5]
```diff
@@ -36,74 +37,78 @@ localization:
...
-  (:guilabel:`O-13` Gran Contribuyente, :guilabel:`O-15` Autorretenedor, :guilabel:`O-23` Agente de
-  retención IVA, :guilabel:`O-47` Regimen de tributación simple, :guilabel:`R-99-PN` No Aplica).
+  (`O-13` Gran Contribuyente, `O-15` Autorretenedor, `O-23` Agente de retención IVA, `O-47` Regimen
+  de tributación simple, `R-99-PN` No Aplica).
 - :guilabel:`Gran Contribuyente`: If the company is *Gran Contribuyente* this option should be
   selected.
-- :guilabel:`Fiscal Regimen`: Select the Tribute Name for the company (:guilabel:`IVA`,
-  :guilabel:`INC`, :guilabel:`IVA e INC`, :guilabel:`No Aplica`)
+- :guilabel:`Fiscal Regimen`: Select the Tribute Name for the company (`IVA`, `INC`, `IVA e INC`,
+  `No Aplica`)
```
Comment (larm-odoo) [inline content/applications/finance/fiscal_localizations/colombia.rst:82]:
```suggestion
 or  `No Aplica`)
```

File: content/applications/finance/fiscal_localizations/colombia.rst [hunk 6]
```diff
@@ -36,74 +37,78 @@ localization:
...
+  company.
 - :guilabel:`Company Registry`: Company's NIT number *without* the verification code.
-- :guilabel:`Account ID`: Company ID followed by `_01`.
-- :guilabel:`Colombia Template Code`: Select one of the two available templates (:guilabel:`CGEN03`
-  or :guilabel:`CGNE04`) to be used in the PDF format of the electronic invoice.
+- :guilabel:`Account ID`: Company's NIT number followed by `_01`.
+- :guilabel:`Colombia Template Code`: Select one of the two available templates (`CGEN03` or
+  `CGNE04`) to be used in the PDF format of the electronic invoice.
+
+Check the :guilabel:`Test mode` checkbox to connect with the Carvajal testing environment.
```
Comment (larm-odoo) [inline content/applications/finance/fiscal_localizations/colombia.rst:101]:
```suggestion
Enable the :guilabel:`Test mode` checkbox to connect with the Carvajal testing environment.
```

File: content/applications/finance/fiscal_localizations/colombia.rst [hunk 7]
```diff
@@ -36,74 +37,78 @@ localization:
...
+
+Once Odoo and Carvajal are fully configured and ready for production, uncheck the :guilabel:`Test
```
Comment (larm-odoo) [inline content/applications/finance/fiscal_localizations/colombia.rst:103]:
```suggestion
Once Odoo and Carvajal are fully configured and ready for production, deactivate the :guilabel:`Test
```

File: content/applications/finance/fiscal_localizations/colombia.rst [hunk 8]
```diff
@@ -36,74 +37,78 @@ localization:
...
    :alt: Configure credentials for Carvajal web service in Odoo.
 
-.. note::
-   Check the :guilabel:`Test mode` checkbox to connect with the Carvajal testing environment. Once
-   Odoo and Carvajal are fully configured and ready for production, uncheck the :guilabel:`Test
-   mode` checkbox to use the production database.
-
 .. important::
-   :guilabel:`Test mode` must be used **only** on replicated databases, **not** the production
+   :guilabel:`Test mode` must be used **only** on duplicated databases, **not** the production
```
Comment (larm-odoo) [inline content/applications/finance/fiscal_localizations/colombia.rst:111]:
```suggestion
   :guilabel:`Test mode` must **only** be used on duplicated databases, **not** the production
```

File: content/applications/finance/fiscal_localizations/colombia.rst [hunk 9]
```diff
@@ -123,67 +137,85 @@ Master data configuration
 Partner
 ~~~~~~~
 
+Partner contacts can be created in the *Contacts* app. To do so, navigate to
```
Comment (larm-odoo) [inline content/applications/finance/fiscal_localizations/colombia.rst:140]:
```suggestion
Partner contacts can be created in the *Contacts* application. To do so, navigate to
```

File: content/applications/finance/fiscal_localizations/colombia.rst [hunk 10]
```diff
@@ -123,67 +137,85 @@ Master data configuration
...
+:menuselection:`Contacts`, and click the :guilabel:`Create` button.
+
+Then, name the contact, and set the contact type by selecting the :guilabel:`Individual` or
+:guilabel:`Company` radio button.
```
Comment (larm-odoo) [inline content/applications/finance/fiscal_localizations/colombia.rst:144]:
```suggestion
Then, name the contact, and using the radio buttons, select the contact type, either :guilabel:`Individual` or :guilabel:`Company`.
```

File: content/applications/finance/fiscal_localizations/colombia.rst [hunk 11]
```diff
@@ -123,67 +137,85 @@ Master data configuration
...
+
+Complete the full :guilabel:`Address`, including the :guilabel:`City`, :guilabel:`State`, and
+:guilabel:`ZIP` code. Proceed to complete the identification and fiscal information.
```
Comment (larm-odoo) [inline content/applications/finance/fiscal_localizations/colombia.rst:147]:
```suggestion
:guilabel:`ZIP` code. Then, complete the identification and fiscal information.
```

File: content/applications/finance/fiscal_localizations/colombia.rst [hunk 12]
```diff
@@ -123,67 +137,85 @@ Master data configuration
...
 
 The required fields can be found under :menuselection:`Partner --> Sales & Purchase Tab --> Fiscal
-Information`:
+Information section`:
 
 - :guilabel:`Obligaciones y Responsabilidades`: Select the fiscal responsibility for the company
-  (:guilabel:`O-13` Gran Contribuyente, :guilabel:`O-15` Autorretenedor, :guilabel:`O-23` Agente de
-  retención IVA, :guilabel:`O-47` Regimen de tributación simple, :guilabel:`R-99-PN` No Aplica).
+  (`O-13` Gran Contribuyente, `O-15` Autorretenedor, `O-23` Agente de retención IVA, `O-47` Regimen
+  de tributación simple, `R-99-PN` No Aplica).
```
Comment (larm-odoo) [inline content/applications/finance/fiscal_localizations/colombia.rst:172]:
```suggestion
  de tributación simple, or `R-99-PN` No Aplica).
```

File: content/applications/finance/fiscal_localizations/colombia.rst [hunk 13]
```diff
@@ -123,67 +137,85 @@ Master data configuration
...
 - :guilabel:`Gran Contribuyente`: If the company is *Gran Contribuyente* this option should be
   selected.
-- :guilabel:`Fiscal Regimen`: Select the Tribute Name for the company (:guilabel:`IVA`,
-  :guilabel:`INC`, :guilabel:`IVA e INC`, :guilabel:`No Aplica`)
+- :guilabel:`Fiscal Regimen`: Select the tribute name for the company (`IVA`, `INC`, `IVA e INC`,
```
Comment (larm-odoo) [inline content/applications/finance/fiscal_localizations/colombia.rst:175]:
```suggestion
- :guilabel:`Fiscal Regimen`: Select the tribute name for the company (`IVA`, `INC`, `IVA e INC`, or
```

File: content/applications/finance/fiscal_localizations/colombia.rst [hunk 14]
```diff
@@ -123,67 +137,85 @@ Master data configuration
...
-:menuselection:`Accounting --> Configuration --> Taxes`, and select the related tax.
+To create or modify taxes, go to :menuselection:`Accounting --> Configuration --> Taxes`, and select
+the related tax.
 
-Retention tax types (:guilabel:`ICA`, :guilabel:`IVA`, :guilabel:`Fuente`) are also included. This
-configuration is used to display taxes in the invoice PDF correctly.
+If sales transactions include products with taxes, the :guilabel:`Value Type` field in the
+:guilabel:`Advanced Options` tab needs to be configured per tax. Retention tax types
+(:guilabel:`ICA`, :guilabel:`IVA`, :guilabel:`Fuente`) are also included. This configuration is used
+to display taxes in the invoice PDF correctly.
```
Comment (larm-odoo) [inline content/applications/finance/fiscal_localizations/colombia.rst:200]:
```suggestion
to display taxes correctly in the invoice PDF.
```

File: content/applications/finance/fiscal_localizations/colombia.rst [hunk 15]
```diff
@@ -259,29 +311,33 @@ invoice is then processed asynchronously by the E-invoicing service UBL 2.1 (Col
...
 
-The electronic invoice vendor (Carvajal) receives the XML file and proceeds to validate its
+The electronic invoice vendor (Carvajal) receives the XML file, and proceeds to validate its
 structure and information.
 
-In the :guilabel:`Action` menu, select the :guilabel:`Check Carvajal Status` button. The
-:guilabel:`Electronic Invoice Status` field value changes to :guilabel:`Validated` if everything
-is correct. Then, proceed to generate a legal XML, which includes a digital signature and a unique
-code (CUFE), and a PDF invoice that includes a QR code, and the previously-generated CUFE code.
+The *electronic invoice status* could be checked in the :guilabel:`Action` menu, by selecting the
```
Comment (larm-odoo) [inline content/applications/finance/fiscal_localizations/colombia.rst:329]:
```suggestion
The *electronic invoice status* can be checked in the :guilabel:`Action` menu, by clicking the
```

File: content/applications/finance/fiscal_localizations/colombia.rst [hunk 16]
```diff
@@ -315,7 +371,7 @@ Debit notes
 
 The process for debit notes is similar to credit notes. To create a debit note with reference to an
 invoice, go to :menuselection:`Accounting --> Customers --> Invoices`. On the invoice, click the
-:guilabel:`Add Debit Note` button and complete the following information:
+:guilabel:`Add Debit Note` button, and complete the following information:
```
Comment (larm-odoo) [inline content/applications/finance/fiscal_localizations/colombia.rst:374]:
```suggestion
:guilabel:`Add Debit Note` button, and enter the following information:
```

**Review by larm-odoo — CHANGES_REQUESTED** [review]
Not a lot of changes, but maybe that's because this was WAY shorter!

**Comment by samueljlieber** [comment]
Thank you @larm-odoo for your review! I implemented most of your suggestions in https://github.com/odoo/documentation/commit/a359dfefabe07cf2877fc389ad941d97ec49f31e. @StraubCreative this PR should be good to go. 👍

**Review by larm-odoo — APPROVED** [review]

**Comment by StraubCreative** [comment]
@robodoo r+

**Pull request closed without merging**